packages/uiweb/src/lib/components/chat/ChatProfile/ChatProfile.tsx

```diff
@@ -85,19 +81,14 @@ const Options = ({ options, setOptions, isGroup, chatInfo, groupInfo, setGroupIn
 
 
 
-export const ProfileHeader: React.FC<IProfileHeader> = ({ chatId }: {chatId: string}) => {
+export const ChatProfile: React.FC<IChatProfile> = ({ chatId, style }: {chatId: string, style?: string}) => {
     const theme = useContext(ThemeContext);
     const { account, env } = useChatData();
     const { getGroupByID } = useGetGroupByID();
```

Comment:
@corlard3y in getgroupbyid hook I think you should add one more dependency of env because of I am on dev and a staging chatid has been passed  and than if I change my env than it won't fetch group info again so hence it won't give us the group info